<launch>
  <node pkg="semantic_camera" type="semantic_camera_node" name="semantic_camera_node" output="screen" />
</launch>
